.github/workflows/ci.yml

```diff
@@ -6,7 +6,8 @@ on:
     branches: [main]
 
 jobs:
-  Format:
+  links:
```

Comment:
```suggestion
  lints:
```

? ;)

Reply:
omfg

Reply:
I was trying to run my favorite web browser, okay???